README.md

```diff
@@ -1,35 +1,466 @@
+
+If your app has in-app purchases, send the IAP event when a user purchases something with real money. The IAP event enables Swrve to build revenue reports for your app and to track the spending habits of your users.
+
+* **In iOS 7+** devices, Apple returns a receipt including multiple purchases. To identify what was  purchased, Swrve needs to be sent the `transactionID` of the item purchased (see `SKPaymentTransaction::transactionIdentifier`).
+
+* **In iOS 6 and lower**, Apple only returns one transaction per receipt, so the Swrve IAP method without the `transactionId` can be used.
+
+* If `paymentProvider` was Apple, use either `Base64EncodedReceipt` or `RawReceipt`, depending on whether the receipt is already encoded. Third party Unity IAP plugins can return the receipt already encoded (and this behavior may be different between iOS7+ receipts and iOS6 receipts).
+
+* For Google Play, the `receipt` and `receiptSignature` should be provided.
+
+* For other platforms (including Amazon) no receipt validation is available.
+
+
+```
+IapRewards rewards = new IapRewards(<rewardCurrency>, <rewardAmount>);
+
+#if UNITY_IPHONE
+if (iOS 7+) {
+  SwrveComponent.Instance.SDK.IapApple(quantity,
+                                       productId,
+                                       <localCost>,
+                                       <localCurrency>,
+                                       rewards,
+                                       Base64EncodedReceipt.FromString(base64Receipt),
+                                       transactionId);
+}
+else { // iOS 6
+  SwrveComponent.Instance.SDK.IapApple(quantity,
+                                       productId,
+                                       <localCost>,
+                                       <localCurrency>,
+                                       rewards,
+                                       Base64EncodedReceipt.FromString(base64Receipt));	
+}
+#elif UNITY_ANDROID
+SwrveComponent.Instance.SDK.IapGooglePlay(productId,
+                                          <localCost>,
+                                          <localCurrency>,
+                                          rewards,
+                                          receipt,
+                                          receiptSignature);
+#else
+// no receipt validation
+SwrveComponent.Instance.SDK.Iap(quantity,
+                                productId,
+                                <localCost>,
+                                <localCurrency>,
+                                rewards);
+#endif
+```
+
+If the purchase is of an item, and not a currency, you can omit the rewards argument:
+```
+SwrveComponent.Instance.SDK.Iap(1, "bird", 9.99, "USD");
+```
+
+The correct Apple bundle ID (iOS) or the Google Server Validation Key (Android) should be added to the integration settings page of the Swrve dashboard. This allows Swrve validate purchases made in the app.
+
+In-app Messaging
+-
+
```

Comment:
Before you can test the in-app message feature in your game, you need to create an In App Message campaign in the Swrve Dashboard.